plugin.video.fanfilm/resources/lib/sources/pl/alltube.py

```diff
@@ -76,9 +76,14 @@ def search(self, title, localtitle, year, search_type):
             titles= []
             titles.append(cleantitle.normalize(cleantitle.getsearch(title)))
             titles.append(cleantitle.normalize(cleantitle.getsearch(localtitle)))
-            
+			
```

Comment:
Na przyszłość, raczej wycinaj puste spacje niż dodawaj ;-)

Reply:
Sorki, będę starał się czyścić.
PS: z jakiego edytora korzystacie, coś co dobrze radzi sobie z whitespace i wcięciami, ja edytuję to w Notepad++ i czasem śmieci zostają.

Reply:
Nie ma sprawy. N++ chyba zły nie jest, tam są jakieś pluginy o ile pamiętam może cos znajdziesz.
Ja używam i polecam ViM, ale to nie jest edytor dla każdego :-D
Całe życie w terminalu... ;-)

Reply:
O dżizas, ja to pod windą wszystko robię. Muszę poszperać.